runtimes/catboost/mlserver_catboost/catboost.py

```diff
@@ -0,0 +1,35 @@
+from catboost import CatBoostClassifier
+
+from mlserver import types
+from mlserver.model import MLModel
+from mlserver.utils import get_model_uri
+from mlserver.codecs import NumpyCodec, NumpyRequestCodec
+
+
+WELLKNOWN_MODEL_FILENAMES = ["model.bst"]
+
+
+class CatboostModel(MLModel):
+    """
+    Implementation of the MLModel interface to load and serve `catboost` models.
+    """
+
+    async def load(self) -> bool:
+        model_uri = await get_model_uri(
+            self._settings, wellknown_filenames=WELLKNOWN_MODEL_FILENAMES
+        )
+
+        self._model = CatBoostClassifier()
```

Comment:
Would there be a need in the future to also support regressor models, etc?

Reply:
indeed, we'll have to introduce a mechanism to choose which class to instantiate, so added a TODO for the regressor and ranker!